packages/widget/src/controllers/transfers/fungible-token-transfer.ts

```diff
@@ -62,6 +68,10 @@ export class FungibleTokenTransferController implements ReactiveController {
   host: ReactiveElement;
   walletContext: ContextConsumer<typeof walletContext, ReactiveElement>;
 
+  get __config(): Config {
```

Comment:
maybe change this for `getSourceNetworkConfig` which will return DomainConfig for selected network?